packages/helpers/src/i18n-create-interpolate-element.js

```diff
@@ -0,0 +1,24 @@
+import { createInterpolateElement } from "@wordpress/element";
+import { sprintf } from "@wordpress/i18n";
+
+/**
+ * Create an interpolated element from a translated string and catches errors.
+ *
+ * @param {string} translatedString The translated string to be interpolated.
+ * @param {[string]} args The arguments to be interpolated into the translated string.
+ * @param {object} conversionMap The conversion map for the interpolated values.
+ *
+ * @returns {object} The interpolated element.
+ */
+export const i18nCreateInterpolateElement = ( translatedString, args, conversionMap ) => {
+	try {
+		return createInterpolateElement(
+			sprintf(
```

Comment:
You changed this to include `sprintf`.
While making the fallback easier, it does add complexity with doing two things.
This means we no longer have the option to _not_ `sprintf` when using this safer translation.
Also, if possible in the future, we could just add the HTML in the translation string? I'm not sure why we always use `sprintf` now. Did you investigate this?

Can you explain a bit your reasoning and thoughts here? Or rather in the issue tech choices? 😬 
I would love some more details on your choices.

Reply:
The breaking error is caused when using sprintf.

Reply:
I highly doubt that, since the source includes a try/catch around it: https://github.com/WordPress/gutenberg/blob/trunk/packages/i18n/src/sprintf.js